Add ability to change settings table name and route (#118)
Laravel-Backpack/Settings: README.md

> ### Security updates and breaking changes
> Please **[subscribe to the Backpack Newsletter](http://backpackforlaravel.com/newsletter)** so you can find out about any security updates, breaking changes or major features. We send an email every 1-2 months.

## Install 

> **Note:** The default table name is `settings`, if you need to change it please read the instruction bellow this general ones.

Reviewer: ```suggestion
**Note:** The default table name is `settings`, if you need to change it please carefully read the comments in the instruction below.
```

> For default setup, write in your terminal:

Reviewer: ```suggestion
In your terminal:
```

php artisan db:seed --class="Backpack\Settings\database\seeds\SettingsTableSeeder"
```

> ### [OPTIONAL] Configuration

Reviewer: I think it would be easier to follow if we include this step as part of the installation instructions, so that those who do need to change the table name don't get to publish everything, run the migration, then have to re-do everything all over again. So maybe something like this instead?

```bash
``` bash
# install the package
composer require backpack/settings

# [optional] change the table name if "settings" is not ok with you
php artisan vendor:publish --provider="Backpack\Settings\SettingsServiceProvider" --tag="config"
# then change the table name in `config/backpack/settings.php`

# publish and run the migration
php artisan vendor:publish --provider="Backpack\Settings\SettingsServiceProvider"
php artisan migrate

# [optional] add a menu item for it to the sidebar_content file
php artisan backpack:add-sidebar-content "<li class='nav-item'><a class='nav-link' href='{{ backpack_url('setting') }}'><i class='nav-icon la la-cog'></i> <span>Settings</span></a></li>"

# [optional] insert some example dummy data to the database
php artisan db:seed --class="Backpack\Settings\database\seeds\SettingsTableSeeder"
```

Reviewer: Feels like step-by-step we're moving towards a difficult setup, that will need a `php artisan backpack:settings:install` 😅

Author: I agree with your changes in the Readme, but I think a install command for this is probably far, or not priority. I don't see many people using this configuration option. But I do see people using spatie settings and backpack settings, because they "complete" each other, one is «entity/model settings» other is a «general setting panel». 

Remember where you saw both of them together ? 🙃 Hope I take some time to finish up that package, I think would make this obsolete and spatie for people working with backpack exclusively. 

For now, I've just addressed your concerns in this PR 👍
